### ISSUE TITLE
Build error using 4.4.0 version

### ISSUE THREAD

**Issue by josefnorlin** [th3rdwave/react-native-safe-area-context#313]
When building with "react-native": "npm:react-native-tvos@0.68.2-6"

We get this warning during build:
Package react-native-safe-area-context has been ignored because it contains invalid configuration. Reason: "dependency.platforms.android.libraryName" is not allowed

Leading to this error:
Invariant Violation: requireNativeComponent: "RNCSafeAreaProvider" was not found in the UIManager.

This happens with react-native-safe-area-context 4.4.0 not with 4.3.4

**Comment by hatem-72** [comment]
Not exactly the same stuff but I also see a new warning during IOS build since the last 4.4.0 version.

```bash
warn Package react-native-safe-area-context contains invalid configuration: "dependency.platforms.android.cmakeListsPath" is not allowed.
Please verify it's properly linked using "react-native config" command and contact the package maintainers about this.
```

**Comment by MuratAkbaba006** [comment]
same issue 😞

**Comment by omuraydin24** [comment]
same here :(

**Comment by darshitDev** [comment]
Please try with `npm i react-native-safe-area-context@4.3.1`
cd iOS; pod install; cd ..

**Comment by BhargaviOS** [comment]
Please try with below steps
	1	npm i react-native-safe-area-context@4.3.1
	2	cd iOS; pod install; cd ..
	3	npm run iOS/android
Hope this will help you

**Comment by faruk-arslan** [comment]
Struggling with the same issue.

**Comment by MuratTastemel** [comment]
only delete ^ in your package json 
"react-native-safe-area-context": "4.3.1", 
and delete lock files , again install it
worked for me

**Comment by janicduplessis** [comment]
Looks like there was an error because some config added is not supported in older cli versions. I just published 4.4.1 to fix it.
